pkg/sources/gcs/gcs.go

```diff
@@ -111,7 +111,7 @@ func (c *persistableCache) shouldPersist() (bool, string) {
 }
 
 // Init returns an initialized GCS source.
-func (s *Source) Init(aCtx context.Context, name string, id int64, sourceID int64, verify bool, connection *anypb.Any, concurrency int) error {
+func (s *Source) Init(aCtx context.Context, name string, id sources.JobID, sourceID sources.SourceID, verify bool, connection *anypb.Any, concurrency int) error {
```

Comment:
nit: same question as jobID - looks like sometimes we use `sourceID` and sometimes it's `sourceId`. should this be standardized or do we use them for different purposes?